content/it/_index.md

```diff
@@ -13,7 +13,7 @@ Il Glossario Cloud Native è un progetto guidato dal Business Value Subcommittee
 
 L'invito a suggerire modifiche, aggiunte e migliorie al Glossario Cloud Native è aperto a chiunque abbia voglia di contribuire. Il processo che adottiamo per lo sviluppo e l'arricchimento di questo dizionario condiviso è guidato dalla community e governato dalla CNCF. Il Glossario fornisce una piattaforma vendor-neutral per organizzare un vocabolario condiviso riguardante le tecnologie cloud native. 
 
-Chiunque voglia contribuire può aprire una issue o una pull request su GitHub. Leggi la sezione [Come Contribuire](/contribuire/) per maggiori informazioni e assicurati di seguire le indicazioni della [Style Guide](/style-guide/).
+Chiunque voglia contribuire può aprire una issue o una pull request su GitHub. Leggi la sezione [Come Contribuire](/it/contribute/) per maggiori informazioni e assicurati di seguire le indicazioni della [Style Guide](/it/style-guide/).
```

Comment:
Dear Italian contributors,
which one would be better here? ("Style Guide" vs. "Guida di Stile")

```suggestion
Chiunque voglia contribuire può aprire una issue o una pull request su GitHub. Leggi la sezione [Come Contribuire](/it/contribute/) per maggiori informazioni e assicurati di seguire le indicazioni della [Guida di Stile](/it/style-guide/).
```

Reply:
Ciao @jihoon-seo! They are both correct, but we can go for Guida di Stile to be more consistent :)
Thank you!

Reply:
@annalisag-spark Okay. 😊

Do you have suggestion for this too?
https://glossary.cncf.io/it/style-guide/#:~:text=style%20guide%20di%20base%20del%20repository%20CNCF
> Il Glossario Cloud Native rispetta la [style guide di base](https://github.com/cncf/foundation/blob/master/style-guide.md) del repository CNCF

Reply:
Yep! Changed into Guida di Stile here too!
Hope I followed the right steps to commit the change and open a PR 😅

Thank you for your meticulous review.